src/ui/wxWidgets/PasswordSafeFrame.h

```diff
@@ -189,7 +193,7 @@ class PasswordSafeFrame : public wxFrame, public Observer
 
 private:
     enum class ViewType { TREE, GRID };
-
+    enum class SortType { GROUP, NAME, DATE };
```

Comment:
SortType --> TreeSortType
for readability/maintainability

Reply:
Done as proposed